doc/api/cli.md

```diff
@@ -758,6 +758,14 @@ added: v7.7.0
 
 Enables the collection of trace event tracing information.
 
+### `--trace-exit`
+<!-- YAML
+added: REPLACEME
```

Comment:
should a version number be specified here?

Reply:
No, `REPLACEME` is used as a placeholder. See https://github.com/nodejs/node/blob/master/doc/releases.md#step-3-update-any-replaceme-and-dep00xx-tags-in-the-docs.